tests/Unit/Pool/accelerator_get_is_peer_api.cpp

```diff
@@ -0,0 +1,29 @@
+#include <hc.hpp>
```

Comment:
@facao missing lit macro. could you help follow the examples of other unit tests and apply them in the tests in this pull request?